packages/ketcher-core/src/domain/entities/sgroup.ts

```diff
@@ -240,7 +240,7 @@ export class SGroup {
     return Vec2.diff(sgroup.pp, sgroup.bracketBox.p1)
   }
 
-  static isSaltOrSolvent(moleculeName: string): boolean {
+  static isSaltOrSolvent(moleculeName?: string): boolean {
```

Comment:
Do we need to make it optional?

Reply:
actually yes we need in the current implementation at least here:
```js
const isDraggingSaltOrSolventOnStructure = SGroup.isSaltOrSolvent(
      possibleSaltOrSolvent?.item.data.name
    )
```

And also it looks like it could be more convenient, because I see several places like that:
```js
!SGroup.isSaltOrSolvent(functionalGroup?.name || '')
```
That is not necessary to use || '', because implementation of the function works well with undefined value.